ngraph_onnx/onnx_importer/utils/conv.py

```diff
@@ -101,15 +102,16 @@ def get_kernel_shape(onnx_node):  # type: (NodeWrapper) -> List[int]
     return kernel_shape
 
 
-def get_strides(onnx_node):  # type: (NodeWrapper) -> List[int]
+def get_strides(onnx_node, kernel_shape=None):  # type: (NodeWrapper, List[int]) -> List[int]
```

Comment:
Let's use `Optional[type]` type annotation for function parameters which may be None.

Reply:
@postrational

Reply:
That's not necessary. Let's omit `Optional` if the default value is `None`, mypy infers the `Optional` itself.